src/recensio/plone/browser/configure.zcml

```diff
@@ -9,6 +9,13 @@
       layer="recensio.plone.interfaces.IRecensioPloneLayer"
       />
 
+  <!-- Control panel -->
+  <browser:page
+      name="recensio-settings"
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      class=".controlpanel.RecensioSettingsControlPanel"
+      permission="cmf.ManagePortal" />
```

Comment:
I would add a layer attribute

Reply:
I would also move this to a `controlpanel` folder.
The `browser` folder will get pretty crowded.